### PR TITLE
Weather features

### PR DESCRIPTION
**Pull request by michalkrzem**
magic

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,6 +5,10 @@ name = "pypi"
 
 [packages]
 requests = "==2.31.0"
+pandas = "*"
```
Comment (jacoor) [inline Pipfile]: It's a good practice to use version lock on packages, like with requests above.
How to find version? It's in lock file or use `pip list` command. 
Please update

File: imgw_api/tests/test_get_imgw_api_data.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import requests
+from pytest import mark
+
+
```
Comment (jacoor) [inline imgw_api/tests/test_get_imgw_api_data.py:5]: I'd mark those tests in name as exploration tests, since you are calling external APIs here.

**Review by jacoor — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (michalkrzem) [reply Pipfile]: I added version numbers

**Review by jacoor — APPROVED** [review]
Nice! Now merge when ready.
The idea is that the creator of the PR should merge it, because he knows if it will not interfere with other work (ie. QA). But that depends from team to team.

**Pull request merged**